### DISCUSSION TITLE
It's getting a bit frustrating...

### DISCUSSION THREAD

**Discussion by Connum** [opentypejs/opentype.js#689] [General]
I know that nobody is obliged to contribute, and I myself had episodes of several months where I couldn't even contribute by reviewing, let alone coding. But with the "requires at least one other maintainer to review in order to merge" rule in place, it's getting more and more frustrating being the only one managing to contribute a bit more frequently. It's in fact one of the reasons I even stopped reviewing more regularly, because when I fix something in somebody else's PR, I'm the last editor so I have to get another contributor on board before anything can get merged...

I guess what I'm trying to say is: I might proceed to ignore outstanding reviews by a second maintainer and merge PRs anyway if the situation doesn't change in the near future. There are now many outstanding PRs with great additions that I'd love to see merged, which will also discourage future possible contributors.

We've also been pushing a new version release out far too long now...

**Comment by ILOVEPIE** [comment]
I'd be happy to do reviews. The problem is that I don't have much time to do actual contributions at the moment because I've got a lot of stuff on my plate.
